assets/css/team.css

```
@@ -1,231 +1,414 @@
+    display: inline-block;
+    margin-top: 0.8rem;
+    padding: 0.6rem 1.2rem;
+    background-color: var(--cv-button-bg);
+    color: var(--cv-button-text) !important;
+    border-radius: 6px;
+    text-decoration: none !important;
+    font-weight: 500;
+    font-size: 1.4rem; /* Increased font size */
+    transition: all 0.3s ease;
+    text-align: center;
+    margin: 1rem auto; /* Increased margin */
```

Biome (1.9.4)

[error] Unexpected shorthand property margin after margin-top



(lint/suspicious/noShorthandPropertyOverrides)